apps/web/public/env-config.js

```diff
@@ -1,3 +1,4 @@
 window._env_ = {
   SKIP_PREFLIGHT_CHECK: 'true',
+  REACT_APP_ENVIRONMENT: 'dev',
```

Comment:
as far as I know, this file is generated from the `.env` and we do override `REACT_APP_ENVIRONMENT` in the deploy pipeline... as far as I see it's already set in the `.env`

Reply:
Yes, totally correct. This file shouldn't be modified.